NAMESPACE

```diff
@@ -44,9 +45,18 @@ export(vaccine_schedule_data_future)
 export(vaccine_schedule_from_data)
 export(vaccine_schedule_future)
 export(vaccine_schedule_scenario)
+importFrom(EpiEstim,estimate_R)
+importFrom(EpiEstim,make_config)
+importFrom(distcrete,distcrete)
+importFrom(epitrix,gamma_mucv2shapescale)
```

Comment:
Please implement this function directly in the package rather than adding as a dependency, as the definition looks trivial